src/table/TableWrapper.jsx

```diff
@@ -49,16 +61,23 @@ export default function TableWrapper(props) {
     return null;
   }
 
+  const tableContainer = useRef();
+
+  useEffect(() => {
+    window.console.log(tableContainer.current.clientWidth, 'container');
```

Comment:
I think it's easier to pass `element` from index. It updates as soon as the size changes, so I think it would work as a dependency for useEffect